src/App.js

```diff
@@ -7,32 +7,19 @@ import playerData from './players.json'
 import resultData from './results.json'
 import { Route, Switch } from 'react-router'
 import { BrowserRouter as Router } from 'react-router-dom'
-import { useEffect, useState } from 'react'
-import saveToLocal from './lib/saveToLocal'
+import useStateWithLocalStorage from './lib/useStateWithLocalStorage'
```

Comment:
Custom hooks are usually stored in a `hooks` folder.
```suggestion
import useStateWithLocalStorage from './hooks/useStateWithLocalStorage'
```